libs/application/templates/social-insurance-administration/income-plan/README.md

```diff
@@ -1,50 +1,46 @@
-# Application Templates Income Plan
+# Income Plan
 
 ## Description
 
-This application template allows for submitting an income plan following an application.
-
-![](./assets/income-plan-flow-chart.png)
+This application template enables the submission of an income plan post-application.
 
 ### States
 
 #### Prerequisites
 
-This state is a temporary state that all new income plans will be created in. It has a short lifespan and is unlisted.
-
-The purpose of this state is to be a guard into the actual income plan. There is an external data step which fetches data from Þjóðskrá and TR. The applicant cannot advance if they have a previous income plan that has been in progress at TR for fewer than 10 days.
+All new income plans are initially in this brief, unlisted state, fetching data from Þjóðskrá and TR. Applicants must wait 10 days if a previous income plan is active at TR.
```

Comment:
@ylfahfa, is this correct? Your last comment got overwritten :speak_no_evil: 
```suggestion
This temporary state is where all new income plans are initially created. It is unlisted and has a short lifespan. It acts as a gatekeeper to the actual income plan, fetching data from Þjóðskrá and TR. Applicants cannot proceed if a previous income plan has been in progress at TR for less than 10 days or if they do not have an active application at TR that requires an income plan.
```

Reply:
I just committed the suggested change